### PR TITLE
[MAINTENANCE] Refactor metadata for ColumnAggregate Expectations

### PR DESCRIPTION
**Pull request by josectobar**
Refactor description into variables for schema generation

- [ ] Description of PR changes above includes a link to [an existing GitHub issue](https://github.com/great-expectations/great_expectations/issues)
- [ ] PR title is prefixed with one of: [BUGFIX], [FEATURE], [DOCS], [MAINTENANCE], [CONTRIB]
- [ ] Code is linted - run `invoke lint` (uses `ruff format` + `ruff check`)
- [ ] Appropriate tests and docs have been updated

For more information about contributing, see [Contribute](https://docs.greatexpectations.io/docs/contributing/contributing_checklist).

After you submit your PR, keep the page open and **monitor the statuses of the various checks made by our continuous integration process at the bottom of the page. Please fix any issues that come up** and [reach out on Slack](https://greatexpectations.io/slack) if you need help. Thanks for contributing!

### CODE REVIEW

File: great_expectations/expectations/core/expect_column_kl_divergence_to_be_less_than.py [hunk 1]
```diff
@@ -63,9 +64,46 @@
 logger = logging.getLogger(__name__)
 logging.captureWarnings(True)
 
+EXPECTATION_SHORT_DESCRIPTION = (
+    "Expect the Kulback-Leibler (KL) divergence (relative entropy) of the specified column "
+    "with respect to the partition object to be lower than the provided threshold."
+)
+PARTITION_OBJECT_DESCRIPTION = (
+    "The expected partition object (see "
+    "[partition_object](https://docs.greatexpectations.io/docs/reference/expectations/distributional_expectations"
```
Comment (NathanFarmer) [inline great_expectations/expectations/core/expect_column_kl_divergence_to_be_less_than.py]: Not sure if the tooltips render markup links like this. This link seems important to have. We should confirm it works though.

File: great_expectations/expectations/core/expect_column_kl_divergence_to_be_less_than.py [hunk 2]
```diff
@@ -813,9 +840,7 @@ def _validate(  # noqa: C901, PLR0912, PLR0915
         return return_obj
 
     @classmethod
-    def _get_kl_divergence_chart(  # noqa: PLR0912,C901 - 13
```
Comment (NathanFarmer) [inline great_expectations/expectations/core/expect_column_kl_divergence_to_be_less_than.py]: Bad merge?

File: great_expectations/expectations/core/expect_column_most_common_value_to_be_in_set.py [hunk 3]
```diff
@@ -31,8 +32,21 @@
...
 
 
+EXPECTATION_SHORT_DESCRIPTION = (
+    "Expect the most common value to be within the designated value set."
+)
+VALUE_SET_DESCRIPTION = "A list of potential values to match."
+TIES_OKAY_DESCRIPTION = (
+    "If True, then the expectation will still succeed if values outside "
+    "the designated set are as common (but not more common) "
+    "than designated values. Default False."
```
Comment (NathanFarmer) [inline great_expectations/expectations/core/expect_column_most_common_value_to_be_in_set.py]: Can you move the "Default False." Back into the docstring? The unchecked box in the UI will indicate this in the forms.

File: great_expectations/expectations/core/expect_column_proportion_of_unique_values_to_be_between.py [hunk 4]
```diff
@@ -36,9 +37,29 @@
     )
     from great_expectations.render.renderer_configuration import AddParamArgs
 
+EXPECTATION_SHORT_DESCRIPTION = (
+    "Expect the proportion of unique values to be between a minimum value and a maximum value."
+)
+MIN_VALUE_DESCRIPTION = (
+    "The minimum proportion of unique values.(Proportions are on the range 0 to 1)"
```
Comment (NathanFarmer) [inline great_expectations/expectations/core/expect_column_proportion_of_unique_values_to_be_between.py]: Just a grammar thing but, "The minimum proportion of unique values (proportions are on the range 0 to 1)."

File: great_expectations/expectations/core/expect_column_proportion_of_unique_values_to_be_between.py [hunk 5]
```diff
@@ -36,9 +37,29 @@
...
+)
+MAX_VALUE_DESCRIPTION = (
+    "The maximum proportion of unique values.(Proportions are on the range 0 to 1)"
```
Comment (NathanFarmer) [inline great_expectations/expectations/core/expect_column_proportion_of_unique_values_to_be_between.py]: "The maximum proportion of unique values (proportions are on the range 0 to 1)."

File: great_expectations/expectations/core/expect_column_quantile_values_to_be_between.py [hunk 6]
```diff
@@ -606,11 +621,15 @@ def _descriptive_quantile_table_renderer(
...
                             "tooltip": {
-                                "content": "expect_column_quantile_values_to_be_between \n expect_column_median_to_be_between"  # noqa: E501
-                                if quantile == 0.50  # noqa: PLR2004
-                                else "expect_column_quantile_values_to_be_between"
+                                "content": (
+                                    "expect_column_quantile_values_to_be_between \n expect_column_median_to_be_between"  # noqa: E501
+                                    if quantile == 0.50  # noqa: PLR2004
+                                    else "expect_column_quantile_values_to_be_between"
+                                )
                             },
```
Comment (NathanFarmer) [inline great_expectations/expectations/core/expect_column_quantile_values_to_be_between.py]: Bad merge?

File: great_expectations/expectations/core/expect_column_unique_value_count_to_be_between.py [hunk 7]
```diff
@@ -37,9 +38,17 @@
     )
     from great_expectations.render.renderer_configuration import AddParamArgs
 
+EXPECTATION_SHORT_DESCRIPTION = (
+    "Expect the number of unique values to be between a minimum value and a maximum value."
+)
+MIN_VALUE_DESCRIPTION = "The minimum number of unique values allowed."
+MAX_VALUE_DESCRIPTION = "The maximum number of unique values allowed."
+SUPPORTED_DATA_SOURCES = ["snowflake", "postgresql"]
```
Comment (NathanFarmer) [inline great_expectations/expectations/core/expect_column_unique_value_count_to_be_between.py]: Incorrect capitalization

**Review by NathanFarmer — CHANGES_REQUESTED** [review]

File: great_expectations/expectations/core/expect_column_quantile_values_to_be_between.py (hunk 6 above)
Comment (josectobar) [reply great_expectations/expectations/core/expect_column_quantile_values_to_be_between.py]: why is this one bad? It just adjusted the string format.

Comment (NathanFarmer) [reply great_expectations/expectations/core/expect_column_quantile_values_to_be_between.py]: I wasn't expecting this part of the code to change, but I guess the formatter only fixes files you are touching.

File: great_expectations/expectations/core/expect_column_kl_divergence_to_be_less_than.py (hunk 1 above)
Comment (josectobar) [reply great_expectations/expectations/core/expect_column_kl_divergence_to_be_less_than.py]: I took it out.

**Review by NathanFarmer — APPROVED** [review]

**Pull request merged**